wpilibc/src/main/native/cpp/smartdashboard/Field2d.cpp

```diff
@@ -57,7 +58,7 @@ FieldObject2d* Field2d::GetObject(std::string_view name) {
       std::make_unique<FieldObject2d>(name, FieldObject2d::private_init{}));
   auto obj = m_objects.back().get();
   if (m_table) {
-    obj->m_entry = m_table->GetEntry(obj->m_name);
+    obj->m_entry = m_table->GetDoubleArrayTopic(obj->m_name).GetEntry({});
```

Comment:
Empty braces?

Reply:
Specifies the default value is an empty array.